rllib/policy/policy.py

```diff
@@ -487,7 +495,7 @@ def compute_single_action(
         return (
             single_action,
             [s[0] for s in state_out],
-            {k: v[0] for k, v in info.items()},
+            tree.map_structure(lambda x: x[0], info),
```

Comment:
nice!

Reply:
just want to point out, we realize ``map_structure()`` maps things recursively.
so the logic here is actually a bit different?

Reply:
yep, that's the point.